src/components/Blog/BlogItem.js

```diff
@@ -0,0 +1,109 @@
+'use client';
+import { useSpring, animated } from '@react-spring/web';
+import Image from 'next/image';
+import Link from 'next/link';
+import React, { useState } from 'react';
+import styles from './BlogList.module.css';
+import Header from '../Header/Header';
+
+const BlogItem = ({ post, empty }) => {
+   const [isHover, setIsHover] = useState(false);
+   const imgStyles = useSpring({
+      config: { mass: 5, tension: 2000, friction: 200 },
+      minHeight: isHover ? '90px' : '0px',
+      boxShadow: isHover
+         ? '0px 0px 105px 3px rgba(46,147,255,0.25)'
+         : '0px 0px 105px 3px rgba(46,147,255,0.00)',
+   });
+
+   if (empty)
+      return (
+         <animated.div
+            onMouseEnter={() => setIsHover(true)}
+            onMouseLeave={() => setIsHover(false)}
+            className={styles.post_container}
+            style={{
+               boxShadow: imgStyles.boxShadow,
+            }}
+         >
+            <div className={styles.content_container}>
+               <div className='flex flex-col items-center justify-center'>
+                  <Header
+                     style={{ marginTop: '10px' }}
+                     title={'No Posts? O.O'}
+                     size={'2.5rem'}
+                     interval={1000}
+                  ></Header>
+                  <div>
+                     No Blog posts match this tag and/or project! . . .{' '}
```

Comment:
Day in the life of a software engineer living in NYC: First I like to start off my day by adding unnecessary spaces after every line of code I write. Then I do pilates.